dxr/plugins/python/indexers.py

```diff
@@ -233,13 +241,13 @@ def analyze_tokens(self):
         utf8_token_gen = tokenize.generate_tokens(
             StringIO(self.contents.encode('utf-8')).readline)
 
-        # These are a mapping from the utf-8 byte starting points provided by
-        # the ast nodes, to the unicode character offset tuples for both the
-        # start and the end points.
+        # This is a mapping from the utf-8 byte starting points provided by
```

Comment:
I'm having trouble understanding what a unicode character offset tuple is. What are the elements of such a tuple? Just start and end? It sounds like start and end are each themselves tuples. I think an example would help me a lot.